ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -31,7 +31,118 @@ CTABLE_BACKENDS = {'SQL': 'ctable.backends.SqlBackend'}
 CTABLE_PREFIX = '{{ localsettings.CTABLE_PREFIX }}'
 {% endif %}
 
-DJANGO_LOG_FILE = os.path.join('..','log',"{{ localsettings.DEPLOY_MACHINE_NAME }}-django.log")
+DJANGO_LOG_FILE = os.path.join('..','log',"{{ localsettings.DEPLOY_MACHINE_NAME }}-commcarehq.django.log")
```

Comment:
changed the name so it resembles what people are used to seeing from the settings.py